src/interchange/src/avro/envelope_debezium.rs

```diff
@@ -84,22 +84,13 @@ impl<'a> AvroDecode for AvroDebeziumDecoder<'a> {
                 }
                 "transaction" => {
```

Comment:
So this identifier will no longer be used, even for Debezium 1.5 ?

Reply:
Correct! In Debezium versions < 1.5, `total_order` (from `transaction`) was insufficient to correctly order/deduplicate rows. In versions >= 1.5, `total_order` is redundant.

[This test](https://github.com/MaterializeInc/materialize/blob/main/test/testdrive/kafka-avro-sources.td#L722) confirms that the new logic doesn't break for Debezium versions < 1.5.

Reply:
I was incorrect! We still need the `total_order` field for updates!